geonode/services/serviceprocessors/wms.py

```diff
@@ -180,7 +180,7 @@ def harvest_resource(self, resource_id, geonode_service):
         geonode_layer.save()
 
     def has_resources(self):
-        return True if len(self.parsed_service.contents) > 1 else False
+        return True if len(self.parsed_service.contents) > 0 else False
```

Comment:
@davisc, any idea why this was compared to `> 1`, instead of `> 0`?

Some of our test endpoints for PKI have only one layer in them (supplied by customer) and can not be registered until this is set to `> 0`.